spiceaidocs/docs/data-connectors/graphql.md

```diff
@@ -265,6 +265,44 @@ sql> select node from stargazers limit 1;
 +------------------+--------+---------------+
 ```
 
-:::warning[Limitations]
-- Automatic object unnesting will overwrite columns if multiple properties contain the same sub-keys.
-:::
\ No newline at end of file
+#### Unnesting Duplicate Columns
+
+By default, the Spice Runtime will error when a duplicate column is detected during unnesting.
+
+For example, this example `spicepod.yml` query would fail due to `name` fields:
+```yaml
+from: graphql:https://localhost
+name: stargazers
+params:
+  unnest_depth: 2
+  json_path: data.users
+  query: |
+    query {
+      users {
+        name
+        metadata {
+          name: email
+        }
+      }
+    }
+```
+
+To avoid this error, you should use [use aliases in your query](https://www.apollographql.com/docs/kotlin/advanced/using-aliases/) where possible. In the example above, we used an alias to induce the duplicate error in our query with `name: email`.
```

Comment:
```suggestion
Avoid this error by [using aliases in the query](https://www.apollographql.com/docs/kotlin/advanced/using-aliases/) where possible. In the example above, a duplicate error in our query was induced with `name: email`.
```

Reply:
Even with these edits this part:
we used an alias to induce the duplicate error in our query with `name: email`.

Is not clear.  There is no concrete example where the naming is the same?  I think inducing it is a little confusing.  Can you reword or provide a different example?

Note the edits are to have a more objective voice.  Try to use this in all documentation.